transaction/examples/package/publish.rtm

```diff
@@ -21,170 +21,170 @@ PUBLISH_PACKAGE
     Tuple(                     # Access Rules Struct
         Map<Enum, Enum>(       # Method auth Field
             Enum(
-                "Native", 
+                "AccessRuleKey::Native", 
                 Enum(
-                    "Package",
-                    Enum("SetRoyaltyConfig")
+                    "NativeFn::Package",
+                    Enum("PackageFn::SetRoyaltyConfig")
                 )
             ), 
             Enum(
-                "AccessRule", 
+                "AccessRuleEntry::AccessRule", 
                 Enum(
-                    "Protected", 
+                    "AccessRule::Protected", 
                     Enum(
-                        "ProofRule", 
+                        "AccessRuleNode::ProofRule", 
                         Enum(
-                            "Require", 
+                            "ProofRule::Require", 
                             Enum(
-                                "StaticNonFungible", 
+                                "SoftResourceOrNonFungible::StaticNonFungible", 
```

Comment:
I actually kinda prefer this syntax to what we had before - it's nicely intentioned 👍

Although you could still use it to lie if you found another well-known string which had the same number... still, better than nothing I think.

I can't wait till we have an inverter so that we can sanity check things.

Reply:
Yep!